components/error_page/error_page.jsx

```diff
@@ -73,6 +75,15 @@ export default class ErrorPage extends React.PureComponent {
                     />
                 </Link>
             );
+        } else if (type === ErrorPageTypes.CHANNEL_NOT_FOUND && isGuest) {
+            backButton = (
+                <Link to={params.get('returnTo')}>
```

Comment:
Probably we want to fix that in the place where the `returnTo` is generated, but maybe we want to use `/` directly here

Reply:
In my testing, the `returnTo` always pointed correctly to the default channel, that's why I didn't consider changing this, but if you have a strong preference for pointing to `"/"`, I'm happy with the change

Reply:
Then I prefer to not change it for now to address the root cause is something goes wrong.